hr_holidays_accrual/models/hr_leave_allocation_accruement.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrLeaveAllocationAccruement(models.Model):
+    """Describes an alteration to the accrual leave balance in order to provide
+    definitive explanation how the actual balance was changing over the time.
+    """
+
+    _name = 'hr.leave.allocation.accruement'
+    _description = 'Leaves Allocation Accruement'
+
+    leave_allocation_id = fields.Many2one(
+        string='Leave Allocation',
+        comodel_name='hr.leave.allocation',
+    )
+    days_accrued = fields.Float(
+        'Number of Days',
+        readonly=True,
```

Comment:
@alexey-pelykh  readonly=True &  required=True?

Reply:
`required` on creation, `readonly` afterwards?

Reply:
This is possible, @nikul-serpentcs

Reply:
@alexey-pelykh @pedrobaeza generally I see in domain.

How user can put data in this fields?

Reply:
For user these fields are readonly, user input is not needed. It's a view-only list

Reply:
AFAIK this is not something for being filled by users, but for ensuring internal data integrity

Reply:
Thanks, @alexey-pelykh @pedrobaeza Noted!! 
This all fields are displayed in the only view purpose.